tests/test_anyhow.rs

```diff
@@ -11,7 +11,7 @@ fn test_anyhow_py_function_ok_result() {
     }
 
     Python::with_gil(|py| {
-        let func = wrap_pyfunction!(produce_ok_result)(py).unwrap();
+        let func = wrap_pyfunction_bound!(produce_ok_result, py).unwrap();
```

Comment:
Is there a reason you changed the calling style here? For consistency across the codebase?

It might be valuable to test both styles somewhere. (Maybe we already do?)

Reply:
Yeah, I'd be happy to change back if desired; I didn't notice whether there are explicit tests of both styles somewhere. TBH I didn't realize the single-arg call to make a closure was even a thing, but in all these cases where the closure is invoked immediately it seems like one call with two args is more clear.